tests/test_users.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from db.seed_db import seed
+from db.connection import connect
+import json 
+
+@pytest.fixture(autouse=True)
+def run_around_tests():
+    seed()
+
+def test_users_register(client):
+    user = {
+        "username":"germanwhip2009",
+        "name":"Jackson French",
+        "email":"whippy95@hotmail.com",
+        "password":"abcdefg"
+    }
+
+    mimetype = 'application/json'
+    headers = {
+        'Content-Type': mimetype,
+        'Accept': mimetype
+    }
+
+    response = client.post('/auth/register', data=json.dumps(user), headers=headers)
+    assert response.status_code == 200
+    assert response.json["user"] == user
```

Comment:
this is a good test for the HTTP behaviour of this endpoint, but it would be good to include some assertions about what changes (if any) have been made to the db as a result of this endpoint being called